vsc_pdf_template_transformer/lib/models/tpl_repeater.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart';
+import 'package:vsc_pdf_template_transformer/utils/widget_json_converter.dart';
+import '../utils/widget_builder.dart' as wb;
+
+part 'tpl_repeater.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplRepeater {
+  String className = 'TplRepeater';
+
+  @WidgetJsonConverter()
+  List<wb.WidgetBuilder?>? array;
```

Comment:
`array` is data only. It provides the array of data to be iterated over. The `childTemplate` will be built for each element  in the array.